src/main/java/listeners/ReactionListener.java

```diff
@@ -26,250 +31,244 @@
+                put(config.getProperty("emoji.green_tick"), () -> {
+                    m.removeReaction(config.getProperty("emoji.indifferent_tick"), user).queue();
+                    m.removeReaction(config.getProperty("emoji.red_tick"), user).queue();
+                });
+                put(config.getProperty("emoji.indifferent_tick"), () -> {
+                    m.removeReaction(config.getProperty("emoji.green_tick"), user).queue();
+                    m.removeReaction(config.getProperty("emoji.red_tick"), user).queue();
+                });
+                put(config.getProperty("emoji.red_tick"), () -> {
+                    m.removeReaction(config.getProperty("emoji.green_tick"), user).queue();
+                    m.removeReaction(config.getProperty("emoji.indifferent_tick"), user).queue();
+                });
+            }};
+
+            map.get(e.getReactionEmote().getEmote().getAsMention()).run();
+
+
             int pro = 0;
             int con = 0;
             int indif = 0;
-            for (MessageReaction reaction : m.getReactions()) {
-                switch (reaction.getReactionEmote().getEmote().getId()) {
-                    case "667450925677543454":
-                        pro = reaction.getCount();
-                        break;
-                    case "667450939208368130":
-                        indif = reaction.getCount();
-                        break;
-                    case "667450953217212436":
-                        con = reaction.getCount();
-                        break;
+
+            for (MessageReaction reaction : m.getReactions()){
+                String emoji = reaction.getReactionEmote().getEmote().getAsMention();
+                if (config.getProperty("emoji.green_tick").equals(emoji)){
+                    pro = reaction.getCount();
+                } else if (config.getProperty("emoji.indifferent_tick").equals(emoji)){
+                    indif = reaction.getCount();
+                } else if (config.getProperty("emoji.red_tick").equals(emoji)){
+                    con = reaction.getCount();
                 }
             }
             List<MessageEmbed> embedList = m.getEmbeds();
-            if (pro >= 6 || con >= 6) {
-                if (!embedList.isEmpty()) {
-                    MessageEmbed embed = embedList.get(0);
-                    EmbedBuilder eb = new EmbedBuilder(embed);
-                    if (embed.getFooter() == null || (embed.getFooter().getText().contains("Approved") && con >= 6) || (embed.getFooter().getText().contains("Rejected") && pro >= 6)) {
-                        eb.setFooter(String.format("Pro: %d, Con: %d => %s", pro, con, pro >= 6 ? "Approved" : "Rejected"));
-                        eb.setTimestamp(LocalDateTime.now(ZoneId.systemDefault()));
-                        m.editMessage(eb.build()).queue();
-                    }
+            if ((pro >= 6 || con >= 6) && !embedList.isEmpty()){
```

Comment:
Why six?

Reply:
6 is 2/3 of the people in that channel, but I changed it to a modifiable number